api/src/libs/githubInvitationInit.ts

```diff
@@ -0,0 +1,162 @@
+import { createAppAuth } from '@octokit/auth'
+import { request } from '@octokit/request'
+import { getGithubPrivateKey } from '../config/githubConfig'
+import { logger } from '@bcgov/common-nodejs-utils';
+
+// cached value
+type APPInitailValue = {
+  initialized: number | null,
+  apps: object,
+  nonInstallatedApp: Function | null,
+}
+
+const  installationApps = <APPInitailValue>{
+  initialized: null,
+  apps: {},
+  nonInstallatedApp: null
+}
+
+/**
+ * getNonInstallationApp
+ * @returns a non installed github app
+ */
+export const getNonInstallationApp = () => {
+    logger.info('getNonInstallationApp')
+  // caches a non installed app
+  try{
+  if (!installationApps.nonInstallatedApp) {
+    const auth = createAppAuth({
+      appId: process.env.GITHUB_APP_ID ||  131752,
```

Comment:
What is `131752`? Try to avoid magic numbers. Maybe declare it in a variable or more likely a constant value.